src/components/controlled/transactions-table/components/transaction-shards/tests/transaction-shards.spec.tsx

```diff
@@ -77,11 +77,11 @@ describe('TransactionShards', () => {
       <mvx-transaction-shards class="custom-class">
           <div class="custom-class transaction-shards">
             <mvx-explorer-link class="transactions-table-body-cell-link" data-testid="${DataTestIdsEnum.shardFromLink}" link="/blocks?shard=0">
-              <span data-testid="${DataTestIdsEnum.senderShard}" slot="content">0</span>
+              <span data-testid="${DataTestIdsEnum.senderShard}" slot="explorer-link-slot">0</span>
```

Comment:
why hardcode?

Reply:
Will be solved in future `Pull Request`. 👍